yaml/xyz/openbmc_project/Provisioning/Provisioning.interface.yaml

```diff
@@ -0,0 +1,30 @@
+name: xyz.openbmc_project.Provisioning
```

Comment:
I don't think 'name' is a recognized field at this level?

Reply:
Also, I wonder if this should be under Network.  The upstream maintainer would have final say anyway I guess.